### PR TITLE
Create Like Unlike Api

### PR DESCRIPTION
**Pull request by ahmediqu**

### CODE REVIEW

File: routes/api/posts.js [hunk 1]
```diff
@@ -102,4 +102,22 @@ router.delete('/:id', auth, async (req, res) => {
     }
 });
 
+// @route  put api/posts/like/:id
+// @desc    like a post
+// @access   private
+router.put('/', auth, async (req, res) => {
```
Comment (streetracer48) [inline routes/api/posts.js]: router.put('/') is work ? I saw you wrote route // @route  put api/posts/like/:id

File: routes/api/posts.js [hunk 2]
```diff
@@ -102,4 +102,22 @@ router.delete('/:id', auth, async (req, res) => {
     }
 });
 
+// @route  put api/posts/like/:id
```
Comment (streetracer48) [inline routes/api/posts.js]: ?

Comment (IqbalAhmedTuhin) [reply routes/api/posts.js]: yes working

File: routes/api/posts.js (hunk 1 above)
Comment (IqbalAhmedTuhin) [reply routes/api/posts.js]: Yes worked

**Pull request merged**